### ISSUE TITLE
Codebases widget on the space dashboard should allow easy access to edit in Che

### ISSUE THREAD

**Issue by serenamarie125** [openshiftio/openshift.io#4069]
Propose design solutions to allow the user to get into Che for that URL (note there is a way to do this now on the code bases page) in the codebases widget.

Currently, you can follow a link to GitHub, but that's not the desired result.  
Investigate different solutions.  One possibility is associating the Che icon with a link which may help educate the user that they will be brought to Che.

Background: The OSIO architects also wanted to see an "edit in che" action off of the codebases widget on the space dashboard

**Comment by serenamarie125** [comment]
@catrobson please review

**Comment by catrobson** [comment]
Additional note:  a user can create multiple workspaces for one codebase.  We'll need to consider how this is handled in the dashboard if you want to open your codebase in Che.

**Comment by AshishDurgude** [comment]
Hi @serenamarie125 and @catrobson 
I will start working on it today (31 July).

**Comment by serenamarie125** [comment]
@AshishDurgude can you please add a link to the design artifacts?

**Comment by muruGanesan** [comment]
@serenamarie125 , @catrobson @AshishDurgude @joshuawilson @bartoszmajsak 
Here is the updated Wireframe:
https://redhat.invisionapp.com/share/5SN9DYO93E7#/313254475_01

**Comment by muruGanesan** [comment]
@joshuawilson, @bartoszmajsak, @aslakknutsen, @serenamarie125 , @gorkem ,  @catrobson 

Updated URL for the UX concept:
https://redhat.invisionapp.com/share/5SN9DYO93E7

**Comment by joshuawilson** [comment]
@catrobson This is something that the Platform team should owns and can work with the other teams as needed. As for a person, @andrewazores has the most recent experience with this.

**Comment by gorkem** [comment]
Looking at the comments, I do not think they are stoppers, we can address them iteratively.  

@joshuawilson I think you have a good point with your comment about recent workspaces can you file that separately.

**Comment by serenamarie125** [comment]
@andrewazores please reach out if you have any questions on the design.  Is this something you are starting soon?

**Comment by andrewazores** [comment]
@serenamarie125 yes, I will probably be taking it on next sprint. I had one remaining question/request in a comment on the Invision design but I'll echo it here: is there an existing (low-res/appropriately sized) image asset for the Che icon that I can reuse?

**Comment by andrewazores** [comment]
Tasks are laid out in OSIO Planner here: https://openshift.io/openshiftio/Openshift_io/plan/detail/503

**Comment by serenamarie125** [comment]
@AshishDurgude @catrobson are you aware if we have the Che icon?

**Comment by muruGanesan** [comment]
@serenamarie125, we need to get the icon from Steven PM

**Comment by gorkem** [comment]
Are you looking for this https://www.eclipse.org/che/images/logo-eclipseche.svg

**Comment by slemeur** [comment]
Let me know if you need something more than the SVG.

**Comment by andrewazores** [comment]
Thanks, I think that SVG should suffice.

**Comment by andrewazores** [comment]
I still have a question outstanding in the Invision comments, regarding the filter behaviour when deep linking from Pipelines back to Codebases: https://redhat.invisionapp.com/share/5SN9DYO93E7#/screens/313254506/comments/102521742 . @AshishDurgude ?
